### ISSUE TITLE
Looking for maintainers.

### ISSUE THREAD

**Issue by LeonardLaszlo** [LeonardLaszlo/nw.js-armv7-binaries#39]
Hello there,

I've built NW.js for ARM when I was a student, because I wanted to run an app on my Odroid.
There was no tutorial at that time and I've mostly asked the help of other people. Thanks again @jtg-gg!

Now, 5 years later, people are expecting me to deliver every version they need. Building 1 version takes more than 24 hours on my laptop. It's not that I don't want to do it, I just have a personal life to live and this is not my priority now. 

TLDR **That is why now I am looking for volunteers to take over the maintenance of this repository.**

Please let me know if anyone is interested.

I wish you the best,
Leonard.

**Comment by LeonardLaszlo** [comment]
I considered running Github actions, but their runner does not have enough resources to build nw.js. 
https://docs.github.com/en/actions/reference/virtual-environments-for-github-hosted-runners

If there is anyone who can donate a runner and maybe setup the whole flow, to make releases on tagging, that would be amazing.

**Comment by hthetiot** [comment]
> I considered running Github actions, but their runner does not have enough resources to build nw.js.

How about Travis-ci may be it provides enough resources.

> TLDR That is why now I am looking for volunteers to take over the maintenance of this repository.

I'm the maintainer of Cordova-plugin-iosrtc and Open-EasyRTC, I'm willing to assist with maintaining that repository. Alternatively, what about trying to convince the official nw.js to add support for ARM binaries?

**Comment by LeonardLaszlo** [comment]
> How about Travis-ci may be it provides enough resources.

The disk space is not enough imo. On 2 cores it will probably run for more than 50 hours. Probably none of the CI providers actually support so long live virtual machines, and most probably they kill them after a while if they don't stop gracefully. But nobody actually documented what is the life span of such a machine.
https://docs.travis-ci.com/user/reference/overview/

> Alternatively, what about trying to convince the official nw.js to add support for ARM binaries?

Sorry, I have better things to do with my life. But I think they know about the possibility of arm builds, since they linked to this Github repo. https://nwjs.io/downloads/

**Comment by jrd261** [comment]
I'm going to reach out to CircleCI to see if they would give us access to an OSS XLarge machine.

**Comment by LeonardLaszlo** [comment]
The automatic build is almost ready, if I find a self-hosted Github runner or a docker host to which I can connect, the build can be scheduled to run weekly.

**Comment by jonwwilkes** [comment]
Did you ever find a machine for a runner?

**Comment by LeonardLaszlo** [comment]
> Did you ever find a machine for a runner?

No. I am running it on my private laptop, and that renders my laptop useless for more than 24h when I do it.

**Comment by ayushmxn** [comment]
I'd be willing to help out. I've created a Github org called NW Userland to consolidate NW tooling. I think moving it there would give it more visibility to it.

Tbh I have a spare laptop but it's not ARM based - and I'm assuming to build NW.js that's what one would need?

**Comment by pd-l2ork** [comment]
I may be able to offer something at my university if I can let me know more about the needs...

**Comment by sysrage** [comment]
@LeonardLaszlo if you have automated builds working, we can find you a GitHub Runner to use. If you move this to the nwutils repo, others can help you maintain it. Please update us on current status. Thank you!
